rero_ils/modules/entities/replace.py

```diff
@@ -0,0 +1,236 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019-2023 RERO
+# Copyright (C) 2019-2023 UCLouvain
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Replace identifiedBy with $ref from MEF."""
+
+
+from copy import deepcopy
+
+import requests
+from datetime import timezone
+from flask import current_app
+
+from rero_ils.modules.documents.api import Document, DocumentsSearch
+from rero_ils.modules.entities.api import Entity
+from rero_ils.modules.entities.utils import init_logger
+from rero_ils.modules.utils import get_mef_url, get_timestamp, \
+    requests_retry_session, set_timestamp
+from datetime import datetime
+
+
+class ReplaceIdentifiedBy(object):
```

Comment:
This class looks like a "Job" or "Task". It could be clearer to reflect this into the classe name.
If a class directly inherit from `object` it's not necessary to declare it.

I propose `class ReplaceIdentifiedByTask:`

It could be great to have a description/usage to this class : describing its purpose, with small exemple.
The docstring `Entity Replace identifiedBy with $ref.` seems a little bit short (my opinion)

Reply:
Was inspierd by `sync/SyncEntity`. This class is used in a task and not a task yourself.